src/Markdig.Tests/Specs/Specs.cs

```diff
@@ -17582,8 +17582,8 @@ public void Example003()
             //     +---------+---------+---------+
             //     | Col1    | Col2    | Col3    |
             //     | Col1a   | Col2a   | Col3a   |
-            //     | Col12             | Col3b   |
-            //     | Col123                      |
+            //     | Col1b             | Col3b   |
```

Comment:
You should not change directly this file but instead modify the [GridTableSpecs.md](https://github.com/lunet-io/markdig/blob/master/src/Markdig.Tests/Specs/GridTableSpecs.md) and then save the `Specs.tt` file from VS to force the code gen of `Specs.cs`